.travis.yml

```diff
@@ -1,12 +1,24 @@
-os:
-- linux
+sudo: false
 language: node_js
-node_js:
-- '8'
-before_install:
-  - npm i -g npm@6.0.0
-cache:
-  directories:
-    - node_modules
+
+matrix:
+  include:
+  - node_js: stable
+    env: CXX=g++-4.8
+
 script:
-  - npm run lint && npm test && npm run build && npm run test:e2e:ci
+  - npm run lint
+  - npm test
```

Comment:
`npm run build` needs to be run prior to the e2e tests so we have the static build to tests against.

Reply:
fixed. thanks!

Reply:
Why is this file not removed instead?